assemblyline_core/dispatching/client.py

```diff
@@ -96,35 +89,20 @@ def outstanding_services(self, sid) -> Dict[str, int]:
         :return: Dictionary of services and number of files
                  remaining per services e.g. {"SERVICE_NAME": 1, ... }
         """
-        # Download the entire status table from redis
-        dispatch_hash = DispatchHash(sid, self.redis)
-        all_service_status = dispatch_hash.all_results()
-        all_files = dispatch_hash.all_files()
-        self.log.info(f"[{sid}] Listing outstanding services {len(all_files)} files "
-                      f"and {len(all_service_status)} entries found")
-
-        output: Dict[str, int] = {}
-
-        for file_hash in all_files:
-            # The schedule might not be in the cache if the submission or file was just issued,
-            # but it will be as soon as the file passes through the dispatcher
-            schedule = dispatch_hash.schedules.get(file_hash)
-            status_values = all_service_status.get(file_hash, {})
-
-            # Go through the schedule stage by stage so we can react to drops
-            # either we have a result and we don't need to count the file (but might drop it)
-            # or we don't have a result, and we need to count that file
-            while schedule:
-                stage = schedule.pop(0)
-                for service_name in stage:
-                    status = status_values.get(service_name)
-                    if status:
-                        if status.drop:
-                            schedule.clear()
-                    else:
-                        output[service_name] = output.get(service_name, 0) + 1
-
-        return output
+        dispatcher_id = self.submission_assignments.get(sid)
+        if dispatcher_id:
+            queue_name = reply_queue_name(prefix="D", suffix="WQ")
```

Comment:
WQ suffix is usually used for work queues for the live UI. Can we pick another suffix for internal dispatcher messages?